suitcase-{{ cookiecutter.subproject_name }}/requirements-dev.txt

```diff
@@ -1,15 +1,19 @@
 # These are required for developing the package (running the tests, building
 # the documentation) but not necessarily required for _using_ it.
+attrs >=18.1.0
```

Comment:
This should also get the `# required in order to import ophyd test fixtures` designation (which I admit I neglected when I first added it to `suitcase-tiff`). Fixed over there in https://github.com/NSLS-II/suitcase-tiff/pull/5/commits/e2c7e6af6b909eeaaf471baebe4e42c1b54c6bbf.